Header header
geometry_msgs/Pose pose
uchile_workspace_analysis/GraspPoint[] grasp
